Adding missed step of logging in to AZS

benjaminfoden opened the pull request:
Added step 6 to login to Azure Stack Environment, otherwise step 7 fails (was step 6).

azurecla commented:
Hi __@Drewm3__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
Trusted source

benjaminfoden committed:
Adding missed step of logging in to AZS

Added step 6 to login to Azure Stack Environment, otherwise step 7 fails (was step 6).
